client/extensions/woocommerce/app/products/index.js

```diff
@@ -81,9 +84,16 @@ class Products extends Component {
 	}
 
 	render() {
-		const { className, site, translate, productsLoading, productsLoaded, totalProducts } = this.props;
+		const { className, site, translate, productsLoading, productsLoaded, totalProducts, productsError } = this.props;
 		const classes = classNames( 'products__list', className );
 
+		if ( productsError ) {
+			return <EmptyContent
+				title={ translate( 'We are having trouble loading your products.' ) }
+				line={ translate( 'Please try again.' ) }
```

Comment:
I wonder what the user will interpret as "trying again". Are they expected to refresh the page in the browser? Or somehow refresh the API call?